### PR TITLE
Clustering

### PR DESCRIPTION
**Pull request by rzuritamilla**
pls review and merge with master

### CODE REVIEW

File: clustering/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Clustering
+Clustering is a popular exploratory data analysis method that allows analysts to study their datasets at a higher level of abstraction.
+Here we use [*k*-means](https://en.wikipedia.org/wiki/K-means_clustering) to identify regions with similar phenology (i.e. *phenoregions*).
+Different kind of *phenoregions* are defined based from the leaf and bloom indices (products of the extended spring indices), and the AVHRR SOST phenological metric. In the near future we will create phenoregions from other indices/metrics and from the combination of both.
+
+The optimal *k* value was identified by the ”elbow” of the [Within Cluster Sum of Squared Error (WCSSE) graph](https://en.wikipedia.org/wiki/Determining_the_number_of_clusters_in_a_data_set).
+
+*Phenoregions* were obtained by applying *k*-means to a matrix with as many rows as valid grid cells in CONUS, and as many columns as years in the database. The SCALA/Python code was embedded in [Jupyter notebooks](https://github.com/phenology/infrastructure/tree/applications/applications/notebooks/stable) and run in a SPARK cluster.
```
Comment (romulogoncalves) [inline clustering/README.md]: Should we had a link to explain it?

File: clustering/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+# Correlation
+The ecological meaning of land surface phenological metrics is not fully clear yet. To shed light on this, we performed a spatio-temporal correlation analysis between the Leaf and Bloom indices and the SOS metric. The first step in the correlation analysis was to deal with the negative values in the AVHRR SOS metric.
+
+From the meta data of the AVHRR phenology products: 
+> The SOST units indicate the day of the year when positive; negative values indicate a SOST day in the previous year (for example, a SOST value of -10 for 2002 indicates a start of season day of 355 in 2001). Valid values range from -150 to 365. In the SOST data layer, a cell value of 1000 represents water bodies and a cell value of -1000 represents the area where a SOST could not be detected due to insufficient change in time-series NDVI or due to lack of sufficient input data
+
+Thus, we first transformed the negative numbers [-150,0) to positive numbers representing the day of year (of year -1)
+Then we calculated the per pixel Pearson correlation between each of the spring indices and the AVHRR SOS value.
```
Comment (romulogoncalves) [inline clustering/README.md]: Is it a new sentence? The previous sentence needs a dot.

**Comment by rzuritamilla** [comment]
all fixed by https://github.com/phenology/hsr-phenological-modelling/commit/b5e11882813f49605c8c969c39cdce0206ec6051

**Comment by romulogoncalves** [comment]
Merge is done

**Pull request merged**